Features/DeleteFeature

```diff
@@ -0,0 +1,10 @@
+@test
+Feature:
+  Verify different Delete operations using Rest-Assured
+
+  Scenario:
```

Comment:
sticking with the tagging discussion, immediately above the scenario line is where you would place scenario-level tags. In this case, you might consider @deleteprofile as the tag used here, which acts specifically on this scenario.  So in this particular file, you start with 3 tags - an application level tag (@servicetests), a feature file level tag (@deletetests) and a scenario-level tag (@deleteprofile).  Once in place, this allows you to have great control over exactly what you are testing.